src/odemis/driver/hamamatsurx.py

```diff
@@ -564,7 +565,7 @@ def terminate(self):
             self.t_image.join(5)
         try:
             self._stop()  # stop any acquisition
-        except Exception:  # TODO which exception?????
+        except Exception:  # TODO which exception????????
```

Comment:
You probably can drop this TODO. This except is just global catch to not fail in `terminate()`.